### PR TITLE
Use real instead of fake focus on table cells

### PR DESCRIPTION
**Pull request by ZarkoPernar**

### CODE REVIEW

File: packages/grid/src/BaseCell.css [hunk 1]
```diff
@@ -54,7 +56,7 @@
   border-color: var(--grid-editableCell-borderColor);
 }
 
-.uitkGridBaseCell-focused {
+.uitkGridBaseCell:focus .uitkGridBaseCell-valueContainer {
```
Comment (joshwooding) [inline packages/grid/src/BaseCell.css:59]: You probably want focus-visible here

Comment (ZarkoPernar) [reply packages/grid/src/BaseCell.css:59]: Good idea

**Review by ararus — APPROVED** [review]

**Review by ararus — DISMISSED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
We need to add a changeset

**Comment by ararus** [comment]
> 

Changeset added

**Review by ararus — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**